src/db/postgres/lrsql/postgres/sql/delete.sql

```diff
@@ -61,3 +61,44 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope::scope_enum;
+
+----------------------begin components of delete-actor-----
+-- :name delete-actor-st2st
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement 
+WHERE ancestor_id IN (query_goes_here)
+OR descendant_id IN (query_goes_here)
```

Comment:
?

Reply:
oops!  yeah, it's the same subquery as anywhere else

Reply:
fixed